### ISSUE TITLE
[dev.icinga.com #591] Scheduling a host for downtime after host is down does not prevent alerts

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga-core#291]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/591

**Created by mfriedrich on 2010-07-08 16:28:34 +00:00**

Assignee: _(none)_
Status: _Closed_  (closed on _2011-07-12 18:45:27 +00:00_)
Target Version: _(none)_
Last Update: _2011-07-12 18:45:27 +00:00 (in Redmine)_

---

http://tracker.nagios.org/view.php?id=35

    Summary      0000035: Scheduling a host for downtime after host is down does not prevent alerts
    Description     Hello,

    Scheduling a host for downtime after the host is already down does not work, the scheduled time shows in the web interface but Nagios still generates host down alerts throughout the downtime period.

    Thanks,

    -greg
    Additional Information  I attached a log file with an example of what we are seeing. Best I can tell this happens every time. I mixed the History log and the Notification log based on time stamps so you can see the problem easier. The older events are at the bottom.

    History Log
    [06-29-2009 05:36:30] HOST DOWNTIME ALERT: wtpnvct1227;STOPPED; Host has exited from a period of scheduled downtime  
    [06-28-2009 21:33:48] HOST ALERT: host1227;UP;HARD;1;PING OK - Packet loss = 0%, RTA = 0.96 ms
    [06-28-2009 21:33:38] SERVICE ALERT: host1227;network: ping;OK;HARD;1;PING OK - Packet loss = 0%, RTA = 1.13 ms
    [06-27-2009 14:02:28] HOST ALERT: host1227;DOWN;HARD;1;(Host Check Timed Out)
    [06-27-2009 12:59:48] HOST ALERT: host1227;DOWN;HARD;1;(Host Check Timed Out)
    [06-27-2009 08:34:58] HOST ALERT: host1227;UNREACHABLE;HARD;1;(Host Check Timed Out)
    [06-27-2009 08:22:18] HOST ALERT: host1227;UNREACHABLE;HARD;1;(Host Check Timed Out)

    Notification Log
    host1227    N/A     HOST DOWN   06-26-2009 20:11:38     contact1    alert-host-by-email-long    (Host Check Timed Out)
    host1227    N/A     HOST DOWN   06-26-2009 20:11:38     contact2    alert-host-by-email-long    (Host Check Timed Out)
    host1227    N/A     HOST DOWN   06-26-2009 20:11:38     contact3    alert-host-by-email-long    (Host Check Timed Out)
    host1227    N/A     HOST DOWN   06-26-2009 19:07:28     contact1    alert-host-by-email-long    (Host Check Timed Out)
    host1227    N/A     HOST DOWN   06-26-2009 19:07:28     contact2    alert-host-by-email-long    (Host Check Timed Out)
    host1227    N/A     HOST DOWN   06-26-2009 19:07:28     contact3    alert-host-by-email-long    (Host Check Timed Out)

    History Log
    [06-26-2009 10:37:23] HOST DOWNTIME ALERT: host1227;STARTED; Host has entered a period of scheduled downtime
    [06-25-2009 17:00:59] HOST ALERT: host1227;DOWN;HARD;5;(Host Check Timed Out)

---

**Relations**:

* relates [#591](https://dev.icinga.com/issues/591)
